module/gui/CaptchaDialog.py

```diff
@@ -0,0 +1,257 @@
+            self.imgAnimation.setCacheMode(QMovie.CacheAll)
+            self.imgAnimation.setDevice(self.imgDataBuffer)
+            if self.imgAnimation.isValid():
+                self.emit(SIGNAL("setMovie(QMovie *)"), self.imgAnimation)
+                self.imgAnimation.start()
+                self.ignoreBtn.show()
+                self.closeBtn.hide()
+                if self.currentResultType == "textual":
+                    self.infoLabel.hide()
+                    self.lineEdit.show()
+                    self.submitBtn.show()
+                    self.imgLabel.unsetCursor()
+                elif self.currentResultType == "positional":
+                    self.infoLabel.setText("Click on the image")
+                    self.infoLabel.show()
+                    self.lineEdit.hide()
+                    self.submitBtn.hide()
+                    self.imgLabel.setCursor(Qt.CrossCursor)
+                else:
+                    errMsg = "setupCaptcha: Unknown resultType '" + unicode(self.currentResultType) + "'"
+                    self.imgAnimation.stop()
+                    self.imgDataBuffer.close()
+            else:
+                self.imgDataBuffer.close()
+                errMsg = "setupCaptcha: Could not load the captcha image (2)"
+        else:
+            self.imgDataBuffer.close()
+            errMsg = "setupCaptcha: Could not load the captcha image (1)"
+
+        if errMsg != None:
+            self.log.error(errMsg)
+            self.emit(SIGNAL("setMovie(QMovie *)"), QMovie())
+            self.infoLabel.setText("*** ERROR ***")
+            self.infoLabel.show()
+            self.lineEdit.hide()
+            self.submitBtn.hide()
+            self.ignoreBtn.hide()
+            self.closeBtn.show()
+
+    def setFree(self):
+        self.currentID = None
+        self.currentResultType = None
+        self.hide()
+        self.processing = False
+
+    def setupNoCaptcha(self):
+        pix = QApplication.style().standardIcon(QStyle.SP_MessageBoxInformation).pixmap(64, 64)
+        self.imgLabel.setPixmap(pix)
+        self.imgLabel.show()
+        self.infoLabel.setText("<b>" + _("There is no captcha waiting.") + "</b>")
+        self.infoLabel.show()
+        self.lineEdit.hide()
+        self.submitBtn.hide()
+        self.ignoreBtn.hide()
+        self.closeBtn.show()
+        self.imgLabel.unsetCursor()
+
+    def setupInteractiveCaptcha(self):
+        pix = QApplication.style().standardIcon(QStyle.SP_MessageBoxInformation).pixmap(64, 64)
+        self.imgLabel.setPixmap(pix)
+        self.imgLabel.show()
+        text =  "<b>"      + _("This captcha must be solved with the pyLoad server's webinterface!") + "</b>"
+        text += "<br><br>" + _("Interactive captchas (e.g. ReCaptcha / I'm not a robot) can't be solved")
+        text += "<br>"     + _("with the pyLoad Client. Only captchas which require text input")
+        text += "<br>"     + _("or a single click on a picture are supported.")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['_' may be undefined, or defined from star imports: PyQt4.QtCore, PyQt4.QtGui](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=567903)